ccm_web/package.json

```diff
@@ -3,19 +3,34 @@
   "version": "1.0.0",
   "description": "",
   "scripts": {
-    "dev:server": "nodemon --inspect=0.0.0.0:9229 --config nmon_config.json server/app.ts",
+    "dev:server": "nodemon --inspect=0.0.0.0:9229 --config nmon_config.json server/src/main.ts",
     "dev:client": "webpack --watch --config webpack/webpack.dev.ts",
     "dev": "npm run dev:server & npm run dev:client",
     "build-client": "webpack --config webpack/webpack.prod.ts",
     "build-server": "tsc",
     "preprod": "npm run build-client && npm run build-server",
-    "prod": "NODE_ENV=production node dist/server/app.js"
+    "prod": "NODE_ENV=production node dist/server/src/main.js",
+    "prebuild": "rimraf dist",
+    "build": "nest build",
+    "start": "nest start",
+    "start:dev": "nest start --watch",
+    "start:debug": "nest start --debug --watch",
```

Comment:
Line 16 and 17 seems to be doing what Line 6 is doing. I see that you have installed `@nest/cli` and `nodemon` packages. Can we support either one of them? So this way less libraries. Anyway we won't be using nest CLI in Prod/Non-prod and just Locally.

Reply:
Is the `package.json` looking better to you now? We could remove `@nestjs/cli` entirely from the `devDependencies` if you want. I think the documentation recommends you install it globally any way. https://docs.nestjs.com/cli/overview#installation

Reply:
I feel we should remove `@nestjs/cli` even as dev dependency since it not used in building the project. It is CLI which make the life easier if want to create a new service or module. I think That info can go in the README more like FYI..

Reply:
Looking at this now. May be also able to remove `schematics`